src/ebonite/ext/flask/build/post_install.j2

```diff
@@ -0,0 +1 @@
+ENV EBONITE_RUN_FLASK=false
```

Comment:
why not in Dockerfile.j2?

Reply:
Do you mean `Dockerfile.j2` from Docker builder or from prebuild? For the former we could not specify this as builder uses generic template for all servers/models. For the latter I decided to make prebuilt images fully Ebonite-agnostic (name and internals) to promote their usage in other contexts. For DockerHub users they should look like alternative to FastAPI images prebuilt by FastAPI author.